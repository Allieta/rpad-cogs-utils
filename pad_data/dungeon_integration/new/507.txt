Hera-Sowilo Descended! - Shining Death Queen-Mythical

stage 0

98 - Mystic Flame Knight @ level 13
Basic attacks (1)

100 - Mystic Ice Knight @ level 13
Basic attacks (1)

102 - Mystic Stone Knight @ level 13
Basic attacks (1)

104 - Mystic Light Knight @ level 13
Basic attacks (1)

108 - Mystic Dark Knight @ level 13
Basic attacks (1)

191 - Pyro Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

193 - Frost Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

195 - Taur Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

197 - Harpie Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

199 - Blood Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

793 - Cyclops @ level 13
Heavy Blow
Player -75% HP (50% chance)
----- HP < 30 -----
Mega Impact
20349 Damage - Deal 150% damage (70% chance)
Heavy Blow
Player -75% HP (50% chance)

stage 1

794 - Thunder Giant, Cyclops @ level 10
Heavy Blow
Player -75% HP (50% chance)
----- HP < 50 -----
----- Execute repeatedly. Turn 1 -----
Raised Weapon
Do nothing (1 time use)
----- Loop to 1 after. Turn 2 -----
Giga Impact
30180 Damage - Deal 200% damage

stage 2

910 - Black Baron @ level 5
Poison Edge
6469 Damage - Change Random 3 to Poison, Deal 85% damage (Preemptive) (1 time use)
----- Execute repeatedly. Turn 1 -----
Chaos Edge
7611 Damage - Change Random to Jammer, Deal 100% damage
----- Loop to 1 after. Turn 2 -----
Poison Edge
6469 Damage - Change Random 3 to Poison, Deal 85% damage (1 time use)

stage 3

98 - Mystic Flame Knight @ level 13
Basic attacks (1)

100 - Mystic Ice Knight @ level 13
Basic attacks (1)

104 - Mystic Light Knight @ level 13
Basic attacks (1)

108 - Mystic Dark Knight @ level 13
Basic attacks (1)

199 - Blood Devil @ level 13
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

200 - Blood Demon @ level 13
Flash Bind
Bind Light cards for 2~4 turns (20% chance)

793 - Cyclops @ level 13
Heavy Blow
Player -75% HP (50% chance)
----- HP < 30 -----
Mega Impact
20349 Damage - Deal 150% damage (70% chance)
Heavy Blow
Player -75% HP (50% chance)

stage 4

1093 - Tornado Holy Dragon @ level 5
Fertile Guard
Voids status ailments for 999 turns (Preemptive)
ESCountdown
Display <COUNTER> and skip turn

stage 5

111 - Vampire Lord @ level 15
----- When player has any buff -----
Reset Wave
Voids player buff effects (50% chance)
----- HP <= 100 -----
Dark World
Blind all orbs on the board (10% chance)
----- HP < 20 -----
Resurrection
Enemy recover 100% hp (70% chance) (1 time use)
Nightmare Slash
23572 Damage - Deal 180% damage (3 hits, 60% each) (70% chance)
Dark World
Blind all orbs on the board (10% chance)

200 - Blood Demon @ level 13
Flash Bind
Bind Light cards for 2~4 turns (20% chance)

stage 6

395 - Love Deity, Chaos Venus @ level 5
Chaotic Defense
Absorb Dark damage for 10 turns (Preemptive)
Love Hurricane
8122 Damage - Deal 80% damage (4 hits, 20% each)
----- HP < 50 -----
----- Turn 1 -----
Temptation
Bind 3 random subs for 2~3 turns
----- Execute repeatedly. Turn 1 -----
Candy &
Change Random to Heal (1 time use)
----- Loop to 1 after. Turn 2 -----
Death
20305 Damage - Deal 200% damage (5 hits, 40% each)

stage 7

478 - Hellray Harpie Demon @ level 7
Dark Bind
Bind Dark cards for 2~4 turns (30% chance)
----- HP < 35 -----
Evil Slash
18190 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)
Dark Bind
Bind Dark cards for 2~4 turns (30% chance)

479 - Helldark Blood Demon @ level 7
Flash Bind
Bind Light cards for 2~4 turns (30% chance)
----- HP < 35 -----
Evil Slash
18446 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)
Flash Bind
Bind Light cards for 2~4 turns (30% chance)

stage 8

649 - Awoken Athena @ level 5
Aegis Sparkle
Absorb combo <= 4 damage for 999 turns (Preemptive) (1 time use)
Shining Spear
8122 Damage - Change Random to Light, Deal 80% damage (60% chance)
----- HP < 50 -----
Storm Punisher
12183 Damage - Deal 120% damage (6 hits, 20% each) (70% chance)
----- HP < 15 -----
----- Turn 1 -----
The end is nigh!
Do nothing
Starburst
101527 Damage - Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 5
Shining Death Queen's Whisper
Bind active skills for 5 turns (Preemptive)
----- Turn 1 -----
Protection of the Gods
Voids status ailments for 999 turns
----- HP <= 100 -----
----- Execute repeatedly. Turn 1 -----
Thunder Arrow
6118 Damage - Bind 1 subs for 2 turns, Deal 50% damage (1 time use)
----- Loop to 1 after. Turn 2 -----
Shining Blast
12236 Damage - Change Dark to Light, Deal 100% damage
----- HP < 50 -----
----- Turn 2 -----
Shining Death Queen's Test
Absorb combo <= 5 damage for 5 turns
ESCountdown
Display <COUNTER> and skip turn

Hera-Sowilo Descended! - Shining Death Queen-Legend

stage 0

98 - Mystic Flame Knight @ level 10
Basic attacks (1)

100 - Mystic Ice Knight @ level 10
Basic attacks (1)

104 - Mystic Light Knight @ level 10
Basic attacks (1)

108 - Mystic Dark Knight @ level 10
Basic attacks (1)

191 - Pyro Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

193 - Frost Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

195 - Taur Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

197 - Harpie Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

199 - Blood Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

793 - Cyclops @ level 10
Heavy Blow
Player -75% HP (50% chance)
----- HP < 30 -----
Mega Impact
15390 Damage - Deal 150% damage (70% chance)
Heavy Blow
Player -75% HP (50% chance)

stage 1

794 - Thunder Giant, Cyclops @ level 7
Heavy Blow
Player -75% HP (50% chance)
----- HP < 50 -----
----- Execute repeatedly. Turn 1 -----
Raised Weapon
Do nothing (1 time use)
----- Loop to 1 after. Turn 2 -----
Giga Impact
20455 Damage - Deal 200% damage

stage 2

910 - Black Baron @ level 3
Poison Edge
3467 Damage - Change Random 3 to Poison, Deal 85% damage (Preemptive) (1 time use)
----- Execute repeatedly. Turn 1 -----
Chaos Edge
4079 Damage - Change Random to Jammer, Deal 100% damage
----- Loop to 1 after. Turn 2 -----
Poison Edge
3467 Damage - Change Random 3 to Poison, Deal 85% damage (1 time use)

stage 3

98 - Mystic Flame Knight @ level 10
Basic attacks (1)

100 - Mystic Ice Knight @ level 10
Basic attacks (1)

104 - Mystic Light Knight @ level 10
Basic attacks (1)

108 - Mystic Dark Knight @ level 10
Basic attacks (1)

199 - Blood Devil @ level 10
----- Turn 1 -----
Anger
Increase damage to 130% for the next 3 turns (40% chance)

200 - Blood Demon @ level 10
Flash Bind
Bind Light cards for 2~4 turns (20% chance)

stage 4

1093 - Tornado Holy Dragon @ level 3
Fertile Guard
Voids status ailments for 999 turns (Preemptive)
ESCountdown
Display <COUNTER> and skip turn

stage 5

111 - Vampire Lord @ level 12
----- When player has any buff -----
Reset Wave
Voids player buff effects (50% chance)
----- HP <= 100 -----
Dark World
Blind all orbs on the board (10% chance)
----- HP < 20 -----
Resurrection
Enemy recover 100% hp (70% chance) (1 time use)
Nightmare Slash
18630 Damage - Deal 180% damage (3 hits, 60% each) (70% chance)
Dark World
Blind all orbs on the board (10% chance)

200 - Blood Demon @ level 10
Flash Bind
Bind Light cards for 2~4 turns (20% chance)

stage 6

395 - Love Deity, Chaos Venus @ level 3
Chaotic Defense
Absorb Dark damage for 10 turns (Preemptive)
Love Hurricane
4353 Damage - Deal 80% damage (4 hits, 20% each)
----- HP < 50 -----
----- Turn 1 -----
Temptation
Bind 3 random subs for 2~3 turns
----- Execute repeatedly. Turn 1 -----
Candy &
Change Random to Heal (1 time use)
----- Loop to 1 after. Turn 2 -----
Death
10883 Damage - Deal 200% damage (5 hits, 40% each)

stage 7

478 - Hellray Harpie Demon @ level 5
Dark Bind
Bind Dark cards for 2~4 turns (30% chance)
----- HP < 35 -----
Evil Slash
12425 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)
Dark Bind
Bind Dark cards for 2~4 turns (30% chance)

479 - Helldark Blood Demon @ level 5
Flash Bind
Bind Light cards for 2~4 turns (30% chance)
----- HP < 35 -----
Evil Slash
12600 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)
Flash Bind
Bind Light cards for 2~4 turns (30% chance)

stage 8

649 - Awoken Athena @ level 3
Aegis Sparkle
Absorb combo <= 4 damage for 999 turns (Preemptive) (1 time use)
Shining Spear
4353 Damage - Change Random to Light, Deal 80% damage (60% chance)
----- HP < 50 -----
Storm Punisher
6530 Damage - Deal 120% damage (6 hits, 20% each) (70% chance)
----- HP < 15 -----
----- Turn 1 -----
The end is nigh!
Do nothing
Starburst
54418 Damage - Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 3
Shining Death Queen's Whisper
Bind active skills for 5 turns (Preemptive)
----- Turn 1 -----
Protection of the Gods
Voids status ailments for 999 turns
----- HP <= 100 -----
----- Execute repeatedly. Turn 1 -----
Thunder Arrow
3279 Damage - Bind 1 subs for 2 turns, Deal 50% damage (1 time use)
----- Loop to 1 after. Turn 2 -----
Shining Blast
6558 Damage - Change Dark to Light, Deal 100% damage
----- HP < 50 -----
----- Turn 2 -----
Shining Death Queen's Test
Absorb combo <= 5 damage for 5 turns
ESCountdown
Display <COUNTER> and skip turn

