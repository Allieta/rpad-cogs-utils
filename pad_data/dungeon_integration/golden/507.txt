Hera-Sowilo Descended! - Shining Death Queen-Mythical

stage 0

98 - Mystic Flame Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
191 - Pyro Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
193 - Frost Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
197 - Harpie Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
195 - Taur Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 10

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

stage 1

794 - Thunder Giant, Cyclops @ level 7

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 49 -----
HP <= 49
Raised Weapon
Do nothing
----- HP <= 49 -----
HP <= 49
Giga Impact
Deal 200% damage

stage 2

910 - Black Baron @ level 3

Poison Edge
Change Random 3 to Poison, Deal 85% damage
----- HP <= 100 -----
HP <= 100
Chaos Edge
Change Random to Jammer, Deal 100% damage

stage 3

98 - Mystic Flame Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
200 - Blood Demon @ level 10

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 4

1093 - Tornado Holy Dragon @ level 3

This monster will only use basic attacks (or computing enemy skills failed)
stage 5

111 - Vampire Lord @ level 12

----- HP <= 100 -----
HP <= 100
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 19 -----
HP <= 19
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 20 -----
HP <= 20
Resurrection
Enemy recover 100% hp
Nightmare Slash
Deal 180% damage (3 hits, 60% each)
----- HP <= 20 -----
HP <= 20
Nightmare Slash
Deal 180% damage (3 hits, 60% each)

200 - Blood Demon @ level 10

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 6

395 - Love Deity, Chaos Venus @ level 3

Chaotic Defense
Absorb Dark damage for 10 turns
----- HP <= 100 -----
HP <= 100
Love Hurricane
Deal 80% damage (4 hits, 20% each)
----- HP <= 49 -----
HP <= 49
Temptation
Bind 3 random subs for 2~3 turns
----- HP <= 49 -----
HP <= 49
Candy &
Change Random to Heal
----- HP <= 49 -----
HP <= 49
Death
Deal 200% damage (5 hits, 40% each)

stage 7

479 - Helldark Blood Demon @ level 5

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

478 - Hellray Harpie Demon @ level 5

----- HP <= 100 -----
HP <= 100
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

stage 8

649 - Awoken Athena @ level 3

Aegis Sparkle
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
HP <= 100
Shining Spear
Change Random to Light, Deal 80% damage
----- HP <= 49 -----
HP <= 49
Storm Punisher
Deal 120% damage (6 hits, 20% each)
----- HP <= 14 -----
HP <= 14
The end is nigh!
Do nothing
----- HP <= 14 -----
HP <= 14
Starburst
Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 3

Shining Death Queen's Whisper
Bind active skills for 5 turns
----- HP <= 100 -----
HP <= 100
Protection of the Gods
Voids status ailments for 999 turns

stage 0

102 - Mystic Stone Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
191 - Pyro Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
193 - Frost Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
195 - Taur Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
197 - Harpie Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 13

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

98 - Mystic Flame Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
stage 1

794 - Thunder Giant, Cyclops @ level 10

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 49 -----
HP <= 49
Raised Weapon
Do nothing
----- HP <= 49 -----
HP <= 49
Giga Impact
Deal 200% damage

stage 2

910 - Black Baron @ level 5

Poison Edge
Change Random 3 to Poison, Deal 85% damage
----- HP <= 100 -----
HP <= 100
Chaos Edge
Change Random to Jammer, Deal 100% damage

stage 3

100 - Mystic Ice Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
98 - Mystic Flame Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
200 - Blood Demon @ level 13

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

199 - Blood Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 13

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

stage 4

1093 - Tornado Holy Dragon @ level 5

This monster will only use basic attacks (or computing enemy skills failed)
stage 5

111 - Vampire Lord @ level 15

----- HP <= 100 -----
HP <= 100
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 19 -----
HP <= 19
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 20 -----
HP <= 20
Resurrection
Enemy recover 100% hp
Nightmare Slash
Deal 180% damage (3 hits, 60% each)
----- HP <= 20 -----
HP <= 20
Nightmare Slash
Deal 180% damage (3 hits, 60% each)

200 - Blood Demon @ level 13

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 6

395 - Love Deity, Chaos Venus @ level 5

Chaotic Defense
Absorb Dark damage for 10 turns
----- HP <= 100 -----
HP <= 100
Love Hurricane
Deal 80% damage (4 hits, 20% each)
----- HP <= 49 -----
HP <= 49
Temptation
Bind 3 random subs for 2~3 turns
----- HP <= 49 -----
HP <= 49
Candy &
Change Random to Heal
----- HP <= 49 -----
HP <= 49
Death
Deal 200% damage (5 hits, 40% each)

stage 7

478 - Hellray Harpie Demon @ level 7

----- HP <= 100 -----
HP <= 100
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

479 - Helldark Blood Demon @ level 7

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

stage 8

649 - Awoken Athena @ level 5

Aegis Sparkle
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
HP <= 100
Shining Spear
Change Random to Light, Deal 80% damage
----- HP <= 49 -----
HP <= 49
Storm Punisher
Deal 120% damage (6 hits, 20% each)
----- HP <= 14 -----
HP <= 14
The end is nigh!
Do nothing
----- HP <= 14 -----
HP <= 14
Starburst
Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 5

Shining Death Queen's Whisper
Bind active skills for 5 turns
----- HP <= 100 -----
HP <= 100
Protection of the Gods
Voids status ailments for 999 turns

Hera-Sowilo Descended! - Shining Death Queen-Legend

stage 0

98 - Mystic Flame Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
191 - Pyro Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
193 - Frost Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
197 - Harpie Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
195 - Taur Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 10

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

stage 1

794 - Thunder Giant, Cyclops @ level 7

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 49 -----
HP <= 49
Raised Weapon
Do nothing
----- HP <= 49 -----
HP <= 49
Giga Impact
Deal 200% damage

stage 2

910 - Black Baron @ level 3

Poison Edge
Change Random 3 to Poison, Deal 85% damage
----- HP <= 100 -----
HP <= 100
Chaos Edge
Change Random to Jammer, Deal 100% damage

stage 3

98 - Mystic Flame Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
200 - Blood Demon @ level 10

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 4

1093 - Tornado Holy Dragon @ level 3

This monster will only use basic attacks (or computing enemy skills failed)
stage 5

111 - Vampire Lord @ level 12

----- HP <= 100 -----
HP <= 100
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 19 -----
HP <= 19
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 20 -----
HP <= 20
Resurrection
Enemy recover 100% hp
Nightmare Slash
Deal 180% damage (3 hits, 60% each)
----- HP <= 20 -----
HP <= 20
Nightmare Slash
Deal 180% damage (3 hits, 60% each)

200 - Blood Demon @ level 10

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 6

395 - Love Deity, Chaos Venus @ level 3

Chaotic Defense
Absorb Dark damage for 10 turns
----- HP <= 100 -----
HP <= 100
Love Hurricane
Deal 80% damage (4 hits, 20% each)
----- HP <= 49 -----
HP <= 49
Temptation
Bind 3 random subs for 2~3 turns
----- HP <= 49 -----
HP <= 49
Candy &
Change Random to Heal
----- HP <= 49 -----
HP <= 49
Death
Deal 200% damage (5 hits, 40% each)

stage 7

479 - Helldark Blood Demon @ level 5

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

478 - Hellray Harpie Demon @ level 5

----- HP <= 100 -----
HP <= 100
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

stage 8

649 - Awoken Athena @ level 3

Aegis Sparkle
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
HP <= 100
Shining Spear
Change Random to Light, Deal 80% damage
----- HP <= 49 -----
HP <= 49
Storm Punisher
Deal 120% damage (6 hits, 20% each)
----- HP <= 14 -----
HP <= 14
The end is nigh!
Do nothing
----- HP <= 14 -----
HP <= 14
Starburst
Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 3

Shining Death Queen's Whisper
Bind active skills for 5 turns
----- HP <= 100 -----
HP <= 100
Protection of the Gods
Voids status ailments for 999 turns

stage 0

102 - Mystic Stone Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
191 - Pyro Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
193 - Frost Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
195 - Taur Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
197 - Harpie Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
199 - Blood Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 13

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

98 - Mystic Flame Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
100 - Mystic Ice Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
stage 1

794 - Thunder Giant, Cyclops @ level 10

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 49 -----
HP <= 49
Raised Weapon
Do nothing
----- HP <= 49 -----
HP <= 49
Giga Impact
Deal 200% damage

stage 2

910 - Black Baron @ level 5

Poison Edge
Change Random 3 to Poison, Deal 85% damage
----- HP <= 100 -----
HP <= 100
Chaos Edge
Change Random to Jammer, Deal 100% damage

stage 3

100 - Mystic Ice Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
98 - Mystic Flame Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
104 - Mystic Light Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
108 - Mystic Dark Knight @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
200 - Blood Demon @ level 13

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

199 - Blood Devil @ level 13

This monster will only use basic attacks (or computing enemy skills failed)
793 - Cyclops @ level 13

----- HP <= 100 -----
HP <= 100
Heavy Blow
Player -75% HP
----- HP <= 29 -----
HP <= 29
Mega Impact
Deal 150% damage
Heavy Blow
Player -75% HP

stage 4

1093 - Tornado Holy Dragon @ level 5

This monster will only use basic attacks (or computing enemy skills failed)
stage 5

111 - Vampire Lord @ level 15

----- HP <= 100 -----
HP <= 100
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 19 -----
HP <= 19
Reset Wave
Voids player buff effects
Dark World
Blind all orbs on the board
----- HP <= 20 -----
HP <= 20
Resurrection
Enemy recover 100% hp
Nightmare Slash
Deal 180% damage (3 hits, 60% each)
----- HP <= 20 -----
HP <= 20
Nightmare Slash
Deal 180% damage (3 hits, 60% each)

200 - Blood Demon @ level 13

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns

stage 6

395 - Love Deity, Chaos Venus @ level 5

Chaotic Defense
Absorb Dark damage for 10 turns
----- HP <= 100 -----
HP <= 100
Love Hurricane
Deal 80% damage (4 hits, 20% each)
----- HP <= 49 -----
HP <= 49
Temptation
Bind 3 random subs for 2~3 turns
----- HP <= 49 -----
HP <= 49
Candy &
Change Random to Heal
----- HP <= 49 -----
HP <= 49
Death
Deal 200% damage (5 hits, 40% each)

stage 7

478 - Hellray Harpie Demon @ level 7

----- HP <= 100 -----
HP <= 100
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

479 - Helldark Blood Demon @ level 7

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

stage 8

649 - Awoken Athena @ level 5

Aegis Sparkle
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
HP <= 100
Shining Spear
Change Random to Light, Deal 80% damage
----- HP <= 49 -----
HP <= 49
Storm Punisher
Deal 120% damage (6 hits, 20% each)
----- HP <= 14 -----
HP <= 14
The end is nigh!
Do nothing
----- HP <= 14 -----
HP <= 14
Starburst
Deal 1000% damage (10 hits, 100% each)

stage 9

1250 - Awoken Hera-Sowilo @ level 5

Shining Death Queen's Whisper
Bind active skills for 5 turns
----- HP <= 100 -----
HP <= 100
Protection of the Gods
Voids status ailments for 999 turns

